app/src/main/java/org/ometa/lovemonster/service/ResponseParser.java

```diff
@@ -196,6 +196,12 @@ private Calendar parseDatetime(@Nullable final String datetime) {
      *      the non-blank value, or null otherwise
      */
     private String parseString(final JSONObject jsonObject, final String attributeName) {
+        // JSONObject will helpfully coerce null to "null" for us.  We don't want that.
+        // http://code.google.com/p/android/issues/detail?id=13830
+        if (jsonObject.isNull(attributeName)) {
```

Comment:
So, I think the issue is that, to work with android unit tests, you need to explicitly include the json jar.  I grabbed the latest version, which I think differs in behavior for the version included in the emulator.

That would explain why the spec passes without the code change, but you see something different in the emulator.